bedrock/mozorg/views.py

```diff
@@ -231,3 +237,41 @@ def get_context_data(self, **kwargs):
     def as_view(cls, **initkwargs):
         cache_timeout = initkwargs.pop("cache_timeout", cls.cache_timeout)
         return cache_page(cache_timeout)(super().as_view(**initkwargs))
+
+
+MEICO_EMAIL_SUBJECT = "MEICO Interest Form"
+MEICO_EMAIL_SENDER = "Mozilla.com <noreply@mozilla.com>"
+MEICO_EMAIL_TO = ["meico@mozilla.com"]
+
+
+@json_view
+def meico_email_form(request):
+    """
+    This form accepts a POST request from future.mozilla.org/meico and will send
+    an email with the data included in the email.
+    """
+    CORS_HEADERS = {
+        "Access-Control-Allow-Origin": "*",
+    }
+
+    if request.method == "OPTIONS":
+        return {}, 200, CORS_HEADERS
+
+    if request.method != "POST":
+        raise jsonview.exceptions.BadRequest("Only POST requests are allowed")
+
+    try:
+        data = json.loads(request.body.decode("utf-8"))
+    except json.decoder.JSONDecodeError:
+        raise jsonview.exceptions.BadRequest("Error decoding JSON")
+
```

Comment:
We should validate and enforce some limits on the data here. Maybe use the Forms framework to validate and clean the incoming data?

Reply:
Added the form validation